docs/maintainers/growing-your-project.md

```diff
@@ -0,0 +1,76 @@
+
+Once you have found a reliable group of contributors, consider adding them to a [Contributor Insight Page](../maintainers/maintainers-guide.md#creating-a-new-contributor-insight-page). It can help you better understand their contributions to your project and the open source ecosystem. Here are some methods you can use to help you analyze its data:
+
+- Monitor contributors to see if they are active enough in your repositories to contribute as a team member.
+- Identify your top contributors to extend an invitation to join a team.
+- Identify active contributors in other repositories who may be interested in joining your team.
+
+If you need more tips on how to pick people that can help grow your project, check out our blog post, ["How to Build Your Open Source Dream Team"](https://dev.to/opensauced/how-to-build-your-open-source-dream-team-a-guide-3i90).
+
+## Recruiting Team Members and Contributors Outside of Your Project
+
+If you've looked at your project's[Repository Insight Page](../features/repo-insights.md) and realize there is no one that could help you, consider recruiting new contributors. To do that, create another repository insight page with projects that are similar to yours. From there, determine which contributors suit your project, add them to a list, and monitor their progress.  
+
+:::tip
+
+Consider checking out our [Understanding Repository Insights Data](../features/repo-insights.md) guide to give you an idea on what to look for when narrowing down your list.
+
+:::
+
+Once you've identified the people you want to ask, reach out to them, providing a clear message and a list of expectations.
+
+After the people from your list has agreed to join your project, create [your team on GitHub](https://docs.github.com/en/organizations/organizing-members-into-teams/creating-a-team) and [sync it](../welcome/faqs.md#sync-your-github-team) to your project's Repository Insight page.
+
+![team sync gif](../../static/gif/team-sync.gif)
+
+## Understanding team participation
+
+To ensure that your open source project reaches it growth strategy goal(s), it is important that your team members actively participate in the project as much as possible. The [Contributors tab](../maintainers/maintainers-guide.md#contributors) in your project's Repository Insights page can be helpful in fostering participation. Here are some strategies you can use this tab to monitor your team member's progress:
+
+- Identify the amount of repositories they contributed to in the Repositories section to see if they are immersed in the project.
+- Take note of the date of their recent contribution in the Last Contributed section to see if it aligns with the number in the Repositories section.
+- Check the level in the Act tab to determine how often they contribute to the project
```

Comment:
- What is "Act"?
- It needs a period at the end of the sentence.
- Inconsistency. You use  the term "section" in the other bullet points, and "tab" here.